src/buffer/out/Row.cpp

```diff
@@ -199,6 +217,19 @@ OutputCellIterator ROW::WriteCells(OutputCellIterator it, const size_t index, co
             {
                 _charRow.DbcsAttrAt(currentIndex) = it->DbcsAttr();
                 _charRow.GlyphAt(currentIndex) = it->Chars();
+
+                if (it->DbcsAttr().IsDbcs())

```

Comment:
This is kind tricky but I need it to handle `dbcs` since I'm in CJK environment myself. A cluster with column set to zero will be ignored for rendering.